elated-egrets/bangazon-internal-site
# Pzc department add

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout pzc-department-add
```

## Steps to test

1. Start virtual environment
1. Nuke DB/Migrations
1. Run:
    ```
    python manage.py makemigrations employee_portal
    python manage.py migrate
    django_data.sh employee_portal faker_factory
    ```
1.Make sure Django server is running
1. run 
    `python manage.py runserver`
1. make sure tests are passing
1. make sure you can add a department

DO NOT MERGE UNTIL WE FIX THE MANY MERGE CONFLICTS!
 

## What you changed
Created add form, tests, changed Departments model to Departments_Model, edited all files that reference Departments model to reflect that change


## What was added to read me
Nothing


## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
